### PR TITLE
fix: updated value not displaying when using reactive forms and editableSearchTerm=true #2

### PR DESCRIPTION
**Pull request by miki995**
fix: updated value not displaying when using reactive forms and editableSearchTerm=true

### CODE REVIEW

**Comment by miki995** [comment]
@anjmao can we merge this one?

**Review by rere252 — APPROVED** [review]

**Review by rere252 — APPROVED** [review]

File: src/ng-select/lib/ng-select.component.spec.ts [hunk 1]
```diff
@@ -50,6 +51,52 @@ describe('NgSelectComponent', () => {
...
+        const itemsList = fixture.componentInstance.select.itemsList;
+        expect(itemsList.items.length).toBe(7);
+        expect(itemsList.items[0]).toEqual(jasmine.objectContaining({
+            label: '0',
+            value: 0
+        }));
+
+        expect(select.selectedItems[0]).toEqual(undefined);
+        select.writeValue(30);
+        expect(select.selectedItems[0].value).toEqual(30);
```
Comment (varnastadeus) [inline src/ng-select/lib/ng-select.component.spec.ts:74]: the assertion is wrong, if you remove your change this test will pass anyway

**Comment by rere252** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions 🎆

Still relevant.

**Comment by MarisaNat** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions 🎆

still relevant.

**Comment by mullery** [comment]
I'm having the same issue. Hoping this pull request will get merged soon...

**Comment by kristintonn** [comment]
Having the same issue.

**Comment by hakla** [comment]
Hi!

We're currently facing the same issue and use the following snippet as a workaround:

```
;(this.select as any)._setSearchTermFromItems()
this.select.detectChanges()
```

Since I don't want to rely on internals, I would like to help out. Is there anything I can do to speed this up (e.g. create a new merge request and improve the tests as was suggested)?

fyi the related issue was automatically closed by the github-actions bot, please reopen it (#1710).

Thanks

**Comment by nchizhov** [comment]
Having the same issue.

**Pull request closed without merging**